src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1400,7 +1514,7 @@ class LineChart extends TemporalXAxisComponent {
             }
     
             this.focus = g.append("g")
-                .attr("transform", "translate(-100,-100)")
+                .attr("transform", "translate(-200,-100)")
```

Comment:
why is this fixed to -200 now?

Reply:
Voronoi hover line was rendering on the chart.  This makes sure it's outside the chart initially.